README.md

```diff
@@ -1,94 +1,62 @@
-1. For the compiled binaries to run, you will need to copy your current TF2 installation to `../game` (relative to your repostiory, outside of it).
-2. To setup debugging, in Visual Studio, select `Client (TF)` as the startup project, then go to its `Properties->Configuration Properties->Debugging`. Set `Command` to your `../game/hl2.exe` binary, the `Command Arguments` to `-steam -game tf -insecure -novid -nojoy -nosteamcontroller -nohltv -particles 1 -noborder -particle_fallback 2 -dev -allowdebug` and `Working Directory` to your game installation folder i.e. `../game/bin`. Note: all the paths here are relative to your copy of the repository (same place where `games.sln` is located), do **not** set these values verbatim.
-3. For server, follow the same procedures but choose the `Server (TF)` project and set the `Command` to `../game/srcds.exe`. The suggested server launch options are `-game tf -console -nomaster -insecure +sv_pure 0 +maxplayers 32 +sv_lan 1 -dev -allowdebug`.
-
-NOTE: Team Comtress 2 is no longer compatible with mastercomfig. Please do not use mastercomfig or any other TF2 config.
-
-See [the Valve dev wiki page](https://developer.valvesoftware.com/wiki/Installing_and_Debugging_the_Source_Code) for another explanation of the last two steps.
-
-Other launch options to consider:
-- `sw` to force windowed mode
-- `-w WIDTH -h HEIGHT` to set the resolution
-- `+map MAPNAME` to automatically launch a map on startup
-
-## Legal
-
-Valve, the Valve logo, Steam, the Steam logo, Team Fortress, the Team Fortress logo, Source, the Source logo are trademarks and/or registered trademarks of Valve Corporation in the U.S. and/or other countries.
-
-Team Comtress 2 is not sponsored, endorsed, licensed by, or affiliated with Valve Corporation.
-
-See license for details.
+[![ko-fi](https://img.shields.io/badge/Support%20me%20on-Ko--fi-FF5E5B.svg?logo=ko-fi&style=flat-square)](https://ko-fi.com/mastercoms)
+[![Liberapay](https://img.shields.io/liberapay/receives/mastercoms.svg?logo=liberapay&style=flat-square)](https://liberapay.com/mastercoms/)
+[![Steam donate](https://img.shields.io/badge/Donate%20via-Steam-00adee.svg?style=flat-square&logo=steam)](https://steamcommunity.com/tradeoffer/new/?partner=85845165&token=M9cQHh8N)
+[![Join the Discord chat](https://img.shields.io/badge/Discord-%23comtress--client-7289da.svg?style=flat-square&logo=discord)](https://discord.gg/CuPb2zV)
+
+# Welcome to Team Comtress 2!
+
+### What *is* TC2?
+
+Team Comtress 2 is a fork of an older version of Team Fortress 2, with a community development team fixing bugs, improving performance, and adding quality of life features, with the goal of having those changes pulled upstream by Valve to the modern game.
+
+### What is TC2 *not*?
+* **TC2 is *not* a "pro mod"**
+While much of the work in TC2 will improve the competitive experience, TC2 does not touch gameplay, balance, or competitive rules.
+* **TC2 is *not* a vision for the "good old days" of TF2**
```

Comment:
Maybe mention https://github.com/NicknineTheEagle/TF2-Base?

Reply:
Admirable project, but I think that's a little bit of a stretch, people aren't expecting that kind of thing from TC2. There *are* some misguided expectations about the other things, though, hence why I'm trying to dispel them here.